packages/app/src/systems/Core/utils/database.ts

```diff
@@ -1,14 +1,14 @@
 import type { Table } from 'dexie';
 import Dexie from 'dexie';
-import type { BridgeAsset } from '~/systems/Bridge';
+import type { Asset } from '~/systems/Assets/services/asset';
 
 export class FuelDB extends Dexie {
-  assets!: Table<BridgeAsset, string>;
+  assets!: Table<Asset, string>;
 
   constructor() {
     super('FuelDB');
     this.version(1).stores({
-      assets: '&address, $symbol',
+      assets: '&name, $symbol',
```

Comment:
Why move out from the address?

Reply:
because now address is inside networks and not on root level.
we are not using this db connection now, will be fixed when we add support to custom assets